packages/headless/src/features/commerce/facets/numeric-facet/numeric-facet-actions.ts

```diff
@@ -57,3 +62,15 @@ export const updateNumericFacetValues = createAction(
     }
   }
 );
+
+export const updateManualNumericFacetRange = createAction(
+  'commerce/facets/numericFacet/updateManualRange',
+  (payload: UpdateManualNumericFacetRangePayload) => {
+    try {
+      validateManualNumericRanges({currentValues: [payload]});
+      return {payload, error: null};
+    } catch (error) {
+      return {payload, error: serializeSchemaValidationError(error as Error)};
+    }
```

Comment:
Why do we need this try/catch logic?

Reply:
It's a copy of the above function; But it's un-necessary in this case. I'll simplify.